catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/ClassificationLevel.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.security.banner.marking;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/** Classification enumeration */
+public enum ClassificationLevel {
+  UNCLASSIFIED("UNCLASSIFIED", "U"),
+  RESTRICTED("RESTRICTED", "R"),
```

Comment:
The rest of these classifications appear to be US ones. Why is Restricted in here? There is no Restricted US classification. There is a NATO one...

Reply:
I have created the following ticket to address this in the future: https://codice.atlassian.net/browse/CAL-388

Reply:
I think the lists selected for this are based on what is defined in ISM. ISM Classification All is defined as R,C,U,S,TS.  I believe there are other examples of NATO markings such as COSMIC.  I think the better part of this capability isn't tied to US markings but instead based on the ISM definitions.  I think this is consistent with the purpose of the Alliance project.

Reply:
Tying it to ISM is fine, but I'm pretty sure that ISM has NATO markings as well (I'm not going to look, so someone can correct me if it doesn't), which aren't in here other than R. It'd make more sense if there was a separate enum just for NATO.